var/spack/repos/builtin/packages/apbs/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Apbs(AutotoolsPackage):
+    """APBS solves the equations of continuum electrostatics
+    for large biomolecular assemblages."""
+
+    homepage = "http://www.poissonboltzmann.org/"
+    url      = "https://downloads.sourceforge.net/project/apbs/apbs/apbs-1.3.0/apbs-1.3-source.tar.gz"
+
+    # Working around race condition
+    parallel = False
+
+    # Version 1.4.2.1 uses cmake. Commented out pending resolution of
+    # https://github.com/spack/spack/pull/12941
```

Comment:
Would it be better to comment out 1.3 and keep 1.4+, or do you need 1.4+?

Reply:
I keep going back and forth on this matter. Re-looking at the documentation for the [dependent package](https://rashmikumari.github.io/g_mmpbsa/Download-and-Installation.html) I could go to 1.4. I'm going to mark this as draft for now.